learn/DOM/009_Nodelist/script.js

```diff
@@ -0,0 +1,20 @@
+const nodeList = document.querySelectorAll("p");
+const test = document.getElementById("test");
+const check = document.querySelectorAll("input[type=checkbox]");
```

Comment:
Mesti di ubah biar si for of nya tau variabel mana yang diambil buat looping

```suggestion
const checkbox = document.querySelectorAll("input[type=checkbox]");
```